app/components/publications/batch-documents-publication-modal.hbs

```diff
@@ -18,14 +18,16 @@
             </th>
             <th>
               {{t "publication-number"}}
-              {{#let (concat "tooltip-trigger-" (unique-id)) as |tooltipTriggerId|}}
-                <AuIcon @icon="circle-info" id={{tooltipTriggerId}} class="au-u-muted" />
-                <EmberTooltip @targetId={{tooltipTriggerId}} @tooltipClass="auk-tooltip" @side="bottom">
-                  <p>
-                    {{t "batch-document-publication-modal-publication-number-info"}}
-                  </p>
-                </EmberTooltip>
-              {{/let}}
+              <AuTooltip @placement="bottom" as |tooltip|>
```

Comment:
Small overflow problem with this one. Possibly solved in next PR
<img width="557" height="368" alt="image" src="https://github.com/user-attachments/assets/5657cba4-1e78-4f86-ac5d-a4dd069096c0" />

Reply:
inherits a whitespace: nowrap from tr

Reply:
This (the faulty text wrapping) is fixed, but the flashing issue was not caused by the inherited CSS.

It seems like the `AuTooltip` component bugs when placed on elements that are SVGs (`Node` type elements). When I check the `floating-ui` code within `ember-appuniversum` I noticed this: 

```
assert(
    `FloatingUI (modifier): No reference element was defined.`,
    referenceElement instanceof HTMLElement,
  );
```

Thus the faulty initialisation and flashing.

I temporarily fixed this by converting them to `AuButton` instances. Todo for myself: add bugfix wthin `ember-appuniversum`.